package.json

```diff
@@ -108,12 +119,14 @@
     "*.md": "yaspeller-ci",
     "*.js": "eslint"
   },
-  "husky": {
+  "husky---DISABLED": {
```

Comment:
Why did you disable Husky? If you want to push some broken code to discuss it, you can add `--no-verify` argument to `git commit -m "message" --no-verify` command.

Reply:
`--no-verify` — good tip, i didn't know!

i haven't used husky before, i was just rashly trying to get my changes committed and onto my github -- even if they are bad! ;)

you see, i have constant paranoia that my hard drive will crash at any moment, and so i get very antsy and sweaty when i have staged changes for more than 15 minutes :D